src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -60,13 +60,13 @@ public void testGetLastTradedPrice() throws Exception {
 
     MvcResult result = this.mvc.perform(
         org.springframework.test.web.servlet.request.MockMvcRequestBuilders
-            .get("/iex/lastTradedPrice?symbols=AAPL")
+            .get("/iex/lastTradedPrice?symbols=FB")
```

Comment:
This still is part of your PR, when it really was part of a prior change. Please merge PR#1, because it will make it much easier to review "just" your new changes